types/materialize-css/tooltip.d.ts

```diff
@@ -0,0 +1,95 @@
+/// <reference path="./common.d.ts" />
+
+declare namespace M {
+    class Tooltip extends Component<TooltipOptions> implements Openable {
+        /**
+         * Get Instance
+         */
+        static getInstance(elem: Element): Tooltip;
+
+        /**
+         * Init Tooltip
+         */
+        static init(els: Element, options?: Partial<TooltipOptions>): Tooltip;
+
+        /**
+         * Init Tooltips
+         */
+        static init(els: MElements, options?: Partial<TooltipOptions>): Tooltip[];
+
+        /**
+         * Show tooltip.
+         */
+        open(): void;
+
+        /**
+         * Hide tooltip.
+         */
+        close(): void;
+
+        /**
+         * If tooltip is open.
+         */
+        isOpen: boolean;
+
+        /**
+         * If tooltip is hovered.
+         */
+        isHovered: boolean;
+    }
+
+    interface TooltipOptions {
+        /**
+         * Delay time before tooltip disappears.
+         * @default 0
+         */
+        exitDelay: number;
+
+        /**
+         * Delay time before tooltip appears.
+         * @default 200
+         */
+        enterDelay: number;
+
+        /**
+         * Can take regular text or HTML strings.
+         * @default null
+         */
+        html: string | null;
```

Comment:
Is there any reason to use `string | null` instead of just `string`?

Reply:
This was actually in the initial code ;)